### PR TITLE
adjust for vault 1.2.2

### PR DESCRIPTION
**Pull request by yossi-lavi**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,6 +26,7 @@ compile: sdk demo_app demo_app_vault_orm demo_app_vault_sdk
 
 .PHONY: download_openapi
 download_openapi:
+	mkdir $(OPENAPI_DIR)
```
Comment (david-sackstein) [inline Makefile:29]: mkdir -p $(OPENAPI_DIR)
otherwise if the dir already exists, make fails.

**Review by david-sackstein — COMMENTED** [review]
I pushed a commit to fix the makefile and two compilation errors.

**Comment by yossi-lavi** [comment]
> I pushed a commit to fix the makefile and two compilation errors.

Thanks

File: piiano-vault-orm-sdk/src/main/java/com/piiano/sdk/TokenApi.java [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
 public class TokenApi {
 
 	public static final String TOKEN_PREFIX = "token_";
-    public static final String REASON = "AppFunctionality";
-    public static final String ADHOC_REASON = "";
-    public static final String DEFAULT_TTL = "";
-    public static final Boolean RELOAD_CACHE = false;
 
-    private final static TokensApi tokensApi;
+    private final static TokensClient tokensClient;
```
Comment (ori-shalom) [inline piiano-vault-orm-sdk/src/main/java/com/piiano/sdk/TokenApi.java:22]: Fix indentation

**Review by ori-shalom — APPROVED** [review]

**Pull request merged**